%YAML:1.0

# folder: "/home/milo/datasets/Unity3D/farmsim/long_C_usv_beacon"
folder: "/home/milo/datasets/Unity3D/farmsim/floor1"
use_stereo: 0
use_imu: 1
use_depth: 1
use_range: 1
pause: 0
playback_speed: 3.0
